apisix/plugins/tencent-cloud-cls/cls-sdk.lua

```diff
@@ -62,8 +62,15 @@ local params_cache = {
 local function get_ip(hostname)
     local _, resolved = socket.dns.toip(hostname)
     local ip_list = {}
-    for _, v in ipairs(resolved.ip) do
-        insert_tab(ip_list, v)
+    if not resolved.ip then
+        -- DNS parsing failure
+        local err = resolved
+        core.log.warn("resolve ip failed, hostname: " .. hostname .. ", error: " .. err)
+        insert_tab(ip_list, "127.0.0.1")
```

Comment:
Why insert `127.0.0.1` here?

Reply:
Why don't you put it into the `send_to_cls` before the `host_ip` is assigned to the `log_group_list`

Reply:
Sorry, my mistake.